services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'stats/:provider/:org/:repo',
+  }
+
+  static examples = [
```

Comment:
`examples` is not how we define the docs for the frontend. LLM slop?

The tutorial covers how to add docs to the frontend by writing an OpenAPI spec for the service endpoint here: https://github.com/badges/shields/blob/master/doc/TUTORIAL.md#44-adding-documentation-to-the-frontend

Reply:
Sorry, this is a oversight on my end. I was trying to understand the tutorial and also used GH Copilot to get stub code. 

Should be more careful! Apologies, I have made the correction.